scripts/benchmarks/refs_orca/requirements.txt

```diff
@@ -1,64 +1,40 @@
-accelerate==0.33.0
-aiohappyeyeballs==2.3.5
-aiohttp==3.10.1
+accelerate==0.34.0
+aiohappyeyeballs==2.4.0
+aiohttp==3.10.5
 aiosignal==1.3.1
-asttokens==2.4.1
 async-timeout==4.0.3
 attrs==24.2.0
-auto_gptq @ git+https://github.com/AutoGPTQ/AutoGPTQ.git@caf343b1826301c15f90e2e119cabd0347acfcdf
 bitsandbytes==0.43.3
-certifi==2024.7.4
+certifi==2024.8.30
 charset-normalizer==3.3.2
-coloredlogs==15.0.1
-comm==0.2.2
-contourpy==1.2.1
-cramjam==2.8.3
+contourpy==1.3.0
 cycler==0.12.1
-datasets==2.20.0
-debugpy==1.8.5
-decorator==5.1.1
+datasets==2.21.0
 dill==0.3.8
 docstring_parser==0.16
 einops==0.8.0
-exceptiongroup==1.2.2
-executing==2.0.1
-fastparquet==2024.5.0
 filelock==3.15.4
 fire==0.6.0
 flash-attn==2.6.3
--e git+https://github.com/foundation-model-stack/fms-acceleration.git@8bc65cd3e0f83a7786e4716216fbe1bea702313b#egg=fms_acceleration&subdirectory=plugins/framework
--e git+https://github.com/foundation-model-stack/fms-acceleration.git@8bc65cd3e0f83a7786e4716216fbe1bea702313b#egg=fms_acceleration_aadp&subdirectory=plugins/attention-and-distributed-packing
--e git+https://github.com/foundation-model-stack/fms-acceleration.git@8bc65cd3e0f83a7786e4716216fbe1bea702313b#egg=fms_acceleration_foak&subdirectory=plugins/fused-ops-and-kernels
--e git+https://github.com/foundation-model-stack/fms-acceleration.git@8bc65cd3e0f83a7786e4716216fbe1bea702313b#egg=fms_acceleration_peft&subdirectory=plugins/accelerated-peft
--e git+https://github.com/foundation-model-stack/fms-hf-tuning.git@a8ab68ffaa0d3b49aeb6753bccfdf807672eba69#egg=fms_hf_tuning
+-e git+https://github.com/foundation-model-stack/fms-acceleration.git@4831b35b3f2d8c8116e55faed3d050a51ef962a0#egg=fms_acceleration&subdirectory=plugins/framework
+-e git+https://github.com/foundation-model-stack/fms-acceleration.git@4831b35b3f2d8c8116e55faed3d050a51ef962a0#egg=fms_acceleration_foak&subdirectory=plugins/fused-ops-and-kernels
+-e git+https://github.com/foundation-model-stack/fms-acceleration.git@4831b35b3f2d8c8116e55faed3d050a51ef962a0#egg=fms_acceleration_peft&subdirectory=plugins/accelerated-peft
```

Comment:
maybe you need to fix the cli before updating the references, because otherwise the commit will be missing